index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <header class="navigation">
+            <a class="logo" href="#">Obote</a>
+            <button type="button" class="hamburger"><img src="./assets/images/menu-icon.png" alt="menu icon"></button>
+        </header>
+    </nav>
+    <main>
+        <section class="profile-container">
+            <h1>Denis Obote</h1>
+            <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </section>
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="skill-stack">
+            <ul class="stack-one">
+                <li><h3>Language</h3></li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <ul class="stack-two">
+                <li><h3>Frameworks</h3></li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>            
+            </ul>
+            <ul class="stack-three">
+                <li><h3>Skill</h3></li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </section> 
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="social">
+            <div class="social-media">
+            <ul class="social-media-icon">
+                <li><a href="#"><img src="./assets/icons/github.png" alt="Github"></a></li>
+                <li><a href="#"><img src="./assets/icons/linkedin.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="./assets/icons/Vector.png" alt="Angellist"></a></li>
+                <li><a href="#"><img src="./assets/icons/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/icons/medium.png" alt="Medium"></a></li>
+            </ul>
+                <a class="arrow" href="#"><img src="./assets/icons/element-right.png" alt="arrow"></a>   
+            </div>   
+            <div class="experience-title">
+                <h2>MY RECENT <br> WORK</h2>
+            </div> 
+             
+        </section>
+        
```

Comment:
- [ ] Kindly remove the extra blank lines and indentations for a cleaner code file.
_Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart._
> You can read about this note and other useful ones [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).